tests/functional/export.bats

```diff
@@ -132,7 +131,7 @@ export LIBGUESTFS_BACKEND=direct
     cd "$STAND_ALONE_EXPORT_DIR"
 
     helpers.run_ok "$LAGOCLI" start
-    dummy_file_inside_content=$("$LAGOCLI" shell "$VM_NAME" "cat /root/dummy_file_inside")
+    dummy_file_inside_content=$("$LAGOCLI" shell "$VM_NAME" "cat /root/dummy_file_inside" | tail -n1)
```

Comment:
cool - though that file is indeed with more than one line? (otherwise it still might fail)